### DISCUSSION TITLE
How to remove the Border in thumbnail section?

### DISCUSSION THREAD

**Discussion by ezzra** [LycheeOrg/Lychee#2130] [Q&A] [answered]
What custom css would I need to remove the border (far right and far left) ?

![image](https://github.com/LycheeOrg/Lychee/assets/37742522/8f6bf538-17db-42f6-ad2e-9247aa7514db)

Its still ok on desktop, but on mobile its really to large with this border :

![image](https://github.com/LycheeOrg/Lychee/assets/37742522/e95c18d9-586a-4451-a090-6e16e4ad0a8a)

**Comment by ildyria** [comment, marked as answer]
You mean the padding left right ?
or the spacing between the pictures ? 

If it is padding left-right, I think that one should be doable by editing
https://github.com/LycheeOrg/Lychee/blob/master/resources/views/components/gallery/view/photo-listing.blade.php#L21

I would suggest to have a look at:
https://tailwindcss.com/docs/margin
https://tailwindcss.com/docs/responsive-design

If you provide us a PR using those, it is likely to be accepted.
Here is our contribution guide: https://lycheeorg.github.io/docs/contributions.html :)
